edge-modules/MessagesAnalyzer/Settings.cs

```diff
@@ -36,17 +40,27 @@ class Settings
                     configuration.GetValue(DeviceIdPropertyName, DefaultDeviceId),
                     excludedModules,
                     configuration.GetValue(WebhostPortPropertyName, DefaultWebhostPort),
-                    configuration.GetValue(ToleranceInMillisecondsPropertyName, DefaultToleranceInMilliseconds));
+                    configuration.GetValue(ToleranceInMillisecondsPropertyName, DefaultToleranceInMilliseconds),
+                    configuration.GetValue<string>(LogAnalyticEnabledName),
+                    configuration.GetValue<string>(LogAnalyticWorkspaceIdName),
+                    configuration.GetValue<string>(LogAnalyticSharedKeyName),
+                    configuration.GetValue<string>(LogAnalyticLogTypeName));
             });
 
-        Settings(string eventHubCs, string consumerGroupId, string deviceId, IList<string> excludedModuleIds, string webhostPort, double tolerance)
+        Settings(string eventHubCs, string consumerGroupId, string deviceId, IList<string> excludedModuleIds, string webhostPort, double tolerance, string logAnalyticsEnabled, string logAnalyticsWorkspaceIdName, string logAnalyticsSharedKeyName, string logAnalyticsLogTypeName)
         {
+            Boolean enaVal;
```

Comment:
nit: bool for consistency.
```suggestion
            bool enaVal;
```